templates/serverpod_templates/projectname_server_upgrade/lib/src/birthday_reminder.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:projectname_server/src/generated/protocol.dart';
+import 'package:serverpod/serverpod.dart';
+
+/// This is a simple example of a future call that logs a birthday reminder.
+///
+/// In a real-world application, you would implement the logic to send a
+/// an email or a push notification to the user.
+class BirthdayReminder extends FutureCall<Greeting> {
+  @override
+  Future<void> invoke(Session session, Greeting? object) async {
```

Comment:
```suggestion
  Future<void> invoke(Session session, Greeting? greeting) async {
```

Perhaps this is more clear?

Reply:
Thats unfortunately not possible due to it overriding the method from FutureCall (dart avoid_renaming_method_parameters)